src/url-loader/fs-url-loader.ts

```diff
@@ -64,4 +66,41 @@ export class FSUrlLoader implements UrlLoader {
     }
     return this.root ? pathlib.join(this.root, pathname) : pathname;
   }
-};
+
+  async listFilesInProject(): Promise<string[]> {
+    return this._listFilesInDir('');
+  }
+
+  async _listFilesInDir(pathFromRoot: string): Promise<string[]> {
+    const files = await new Promise<string[]>((resolve, reject) => {
+      fs.readdir(
```

Comment:
There are only two non-test files where we use fs. This one, and generate-elements, and generate-elements could use require, as it's loading a json file from disk that's packaged with the analyzer's code. I don't think that's worth the overhead of another dependency (yet), even if it is just 200KiB